packages/block-editor/src/components/block-controls/use-has-block-controls.js

```diff
@@ -2,34 +2,27 @@
  * WordPress dependencies
  */
 import { __experimentalUseSlotFills as useSlotFills } from '@wordpress/components';
-import warning from '@wordpress/warning';
 
 /**
  * Internal dependencies
  */
 import groups from './groups';
 
+/**
+ * We only care about the "other" slot here.
+ * This check is specifically for allowing the Replace
+ * <MediaReplaceFlow /> on featured images and images
+ * within content locked areas.
+ * https://github.com/WordPress/gutenberg/pull/53410
+ *
+ * TODO: Remove this hook, as having a toolbar with only a Replace button is a
+ * misuse of the toolbar.
+ */
 export function useHasAnyBlockControls() {
```

Comment:
The name of the hook, if the change remains, should be updated to something like `useHasOtherBlockControls`.

Reply:
I considered it too. You're right, the name should get updated if we go with this change.